lib/handlers.js

```diff
@@ -12,18 +12,17 @@ function executeHandler (
 function getHandler (
   module /* : string */,
   method /* : string */
-) /* : Function | void */ {
-  let handler
+) /* : Promise<Function | void> */ {
```

Comment:
Only to avoid throwing errors. I am very open to opinions on the following:
```js
function syncError () {
  throw new Error('SYNC ERROR')
}
Promise.resolve()
  .then(() => syncError())
  .catch((err) => err.message === 'SYNC ERROR')

// vs

function asyncError () {
  return Promise.reject(new Error('ASYNC ERROR'))
}
asyncError()
  .catch((err) => err.message === 'ASYNC ERROR')
```